client/my-sites/importer/author-mapping-pane.jsx

```diff
@@ -181,4 +171,13 @@ class AuthorMappingPane extends React.PureComponent {
 	}
 }
 
-export default localize( AuthorMappingPane );
+const withTotalUsers = ( Component ) => ( props ) => {
+	const { siteId } = props;
+	const { data } = useUsersQuery( siteId, {}, { refetchOnWindowFocus: false } );
```

Comment:
In the previous version of the code, we were specifically fetching up to 50 users. With the current default of `useUsersQuery`, we'll now be fetching 100. Is that change intentional?

Reply:
I opted for default params so it would reuse existing queries cache; `<AuthorSelector />` is used on the same page and also uses `useUsersQuery` now. Although `number` isn't used to construct the `cacheKey` so that assumption might be wrong. Nevertheless, there's no need to fetch more than 1 user as we're only interested in `total`.

Good call though, I've to verify that behaviour.

Reply:
>  Nevertheless, there's no need to fetch more than 1 user as we're only interested in total.

That sounds interesting, and my expectation is that it would make the API response even faster, so it sounds like an interesting option! Let me know how it goes once you try it.